example/package.json

```diff
@@ -0,0 +1,10 @@
+{
+	"scripts": {
+		"start": "aiur --watch ./dist",
+		"compile": "aiur ./dist"
+	},
+	"dependencies": {
+		"aiur": "..",
+		"prismjs": "^1.15.0"
```

Comment:
Users of the package won't need to do that... but for this example it is necessary due to the `..` package in the line above.

Reply:
You mean because installing a local package like that doesn't pull in its dependencies? What about all the other dependencies then?

Might be easier (as in more straightforward for end users) to reference aiur-the-npm-package instead of trying to optimize via a local path. For my part, I tend to copy such examples into a separate directory anyway (as in `npm install foo; cp -R node_modules/foo/sample ./`), which wouldn't work here.

Reply:
We only need this one dependency here, because we import the CSS from that package and assume that it is located in the node_modules folder.

The optimization to the local path means that I can try out changes via that example folder. When I want to try out if file watching works as expected, it is very easy to just cd in that folder and try it out.